lib/interpret_and_optimize/services/pb_shared_aggregation_service.dart

```diff
@@ -43,9 +47,14 @@ class PBSharedInterAggregationService {
   void gatherSharedParameters(
       PBSharedMasterNode sharedMasterNode, PBIntermediateNode rootChildNode) {
     for (var prop in sharedMasterNode.overridableProperties) {
+      var targetUUID = _findLastOf(prop?.UUID, '/');
       prop.value = PBIntermediateNodeSearcherService.searchNodeByUUID(
-          rootChildNode, prop?.UUID);
-      if (prop.type == PBSharedInstanceIntermediateNode) {
+          rootChildNode, targetUUID);
+      if (prop.value == null) {
+        // add Designer Warning here, not even sure if this is the designers fault or not
+        // log.warning('UUID: ${targetUUID} not found in searchNodeByUUID');
```

Comment:
I think we should uncomment this log for now, so we know when it happens again. Then when the refactor hits, we can decide to which part of the logs should go.